torch/optim/adadelta.py

```diff
@@ -201,11 +213,15 @@ def _multi_tensor_adadelta(params: List[Tensor],
                            lr: float,
                            weight_decay: float,
                            rho: float,
-                           eps: float):
+                           eps: float,
+                           maximize: bool):
 
     if len(params) == 0:
         return
 
+    if maximize:
+        grads = torch._foreach_neg(grads)
```

Comment:
Why not try to achieve this by modifying one of the scalars (e.g. lr)?

Reply:
The momentum and statistics would be wrong. Flipping the gradients is the way to go in general for these based on previous discussions we had.